mvpa2/algorithms/group_clusterthr.py

```diff
@@ -294,6 +355,11 @@ def featuresegment_producer(ncols):
         # Step 2: threshold all NULL maps and build distribution of NULL cluster
         #         sizes
         #
+        # TODO: yoh: note that the same bcombos which were used to estimate thrmap
+        #       will now be used to estimate cluster sizes/metric.
+        #       Not sure if such double treatment of the same data doesn't provide
+        #       any bias.  We could  a) split chunks pool into two  b) create new
+        #       pool of bcombos for metric estimation
```

Comment:
There is no double treatment. Usually one would generate the bootstrapped maps, threshold them, and determine cluster sizes. It is only done in two steps to avoid having to keep them and dramatically reduce the memory consumption.